pkgs/servers/tailscale/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub, makeWrapper, iptables, iproute }:
+
+buildGoModule rec {
+  pname = "tailscale";
+  version = "0.97-0";
```

Comment:
As mentioned on IRC, LGTM after switching to the official upstream release (`v0.97`).

Reply:
Done. However, just flagging that this will be a problem for future releases. Upstream, I'm about to mark 0.97-40 as stable, which will be the version number burned into the binaries. `v0.97` doesn't appear anywhere in the software itself.

Fortunately, Nix itself doesn't care about the version number, so I'm happy with whatever is acceptable to nixpkgs for now, until we get to a release that's unrepresentable.

Reply:
(as I tried to explain on IRC, the versioning logic of the tailscale repo is currently a bit broken, and the version names are based on the state of the non-OSS repo. So even though the code itself is all open, the releases end up pointing to an opaque commit hash right now.

I'm working on fixing that so that OSS build versions are entirely based on the OSS tree, but it's not done yet. So, just a heads up that the versioning for this package will change in the future, once all this is fixed upstream)